src/core/jit/synth/synth.rs

```diff
@@ -288,10 +295,14 @@ mod tests {
 
     const CONFIG: &str = include_str!("../fixtures/jsonplaceholder-mutation.graphql");
 
-    fn init<Value: JsonLikeOwned + DeserializeOwned + Serialize + Clone>(
+    fn extend_lifetime_str<'b>(r: &'b str) -> &'static str {
+        unsafe { std::mem::transmute::<&'b str, &'static str>(r) }
+    }
```

Comment:
find an alternative strategy so that unsafe isn't required.

Reply:
can we make try_map some external function then?